packages/puppeteer-core/src/api/Page.ts

```diff
@@ -3174,6 +3178,38 @@ export abstract class Page extends EventEmitter<PageEvents> {
    * method is only available in Chrome.
    */
   abstract openDevTools(): Promise<Page>;
+
+  /**
+   * Disable simulated Bluetooth adapter.
+   * https://webbluetoothcg.github.io/web-bluetooth/#bluetooth-disableSimulation-command
+   *
+   * @experimental
+   */
+  abstract disableBluetoothSimulation(): Promise<void>;
```

Comment:
let's create a Bluetooth class and expose it via `page.bluetooth.doX()` (similar to a11y and tracing)? This would let the page API be smaller and re-use implementations between BiDi, CDP, BiDi+ more easily I think.